packages/components/src/molecules/Card/Card.tsx

```diff
@@ -0,0 +1,61 @@
+import type { HTMLAttributes } from "react";
+import React, { forwardRef } from 'react'
+
+import { Icon } from '../..'
+
+export interface CardProps extends Omit<HTMLAttributes<HTMLDivElement>, 'role'> {
+    /**
+    * Specifies the text to be loaded into the header.
+    */
+    title?: string
+    /**
+    * ID to find this component in testing tools (e.g.: cypress, testing library, and jest).
+    */
+    testId?: string
+    /**
+    * Specifies the card max width, if not informed it will be of 300px.
+    */
+    maxWidth?: string
+    /**
+    * Specifies what icon to use following the FastStore Icon sheet.
+    */
+    iconName?: string
+    /**
+    * Specifies the icon's color
+    */
+    iconColor?: string
+    /**
+    * Function that should be executed when the icon is clicked
+    */
+    iconAction?: () => void
+}
+
+const Card = forwardRef<HTMLDivElement, CardProps>(function Card({
+    title,
+    children,
+    maxWidth,
+    testId = 'fs-card',
+    iconName,
+    iconColor,
+    iconAction
+}, ref) {
+    return (
+        <div data-testid={testId} ref={ref} data-fs-card style={{ maxWidth: `${maxWidth}` }}>
```

Comment:
**suggestion:**  How about using <section> instead of <div> for this wrapper?  Since represents a distinct section of content would be a good choice.

Reply:
I think it's a good idea, updated it